examples/lock-unlock-rewrite/values.localhost.yaml

```diff
@@ -1,16 +1,16 @@
 image:
   repository: lock-unlock/rewrite
   pullPolicy: Never
-  tag: "0.1.0"
+  # tag: "0.1.1"
```

Comment:
Was het je bedoeling om de `tag`s uit te commenten?

Reply:
Die zitten toch al in `values.yaml` ... 🤔

Reply:
'k Heb ze nu weer toegevoegd ... want idd ben je dan wel afhankelijk van dezelfde versie lokaal beschikbaar hebben - dus lokaal gebouwd mét precies dát label - als de released versions ...

Reply:
Dat is waar, al moeten ze nu ook precies overeen komen met de tag die in de README is genoemd. We zouden beiden nog kunnen vervangen met `latest`, dan hoeven we niet met elke nieuwe versie `examples/` te updaten 😉